dashboard/src/features/container/routes.js

```diff
@@ -0,0 +1,20 @@
+import Container from './components/Container'
+
+import { getAuthenticationStatus } from 'features/authn/actions'
+import { fetchCoreInfo } from 'features/core/actions'
+import { fetchTestnetInfo } from 'features/testnet/actions'
+
+const CORE_POLLING_TIME = 2 * 1000
+const TESTNET_INFO_POLLING_TIME = 30 * 1000
+
+export default (store) => ({
```

Comment:
We never got to finish the dashboard refactoring, but I wonder if there's a chance to make this new component look like what we want. Here's what I think we discussed in the past:

1. Explicitly name your exported variables, so that it has the same identifier that is likely to be used in the importing file (in this case, `makeRoutes()`).
2. If possible, naming your files to be the same as the exported identifier (in this case, `makeRoutes.js`).
3. Avoiding the use of index files, so there is only path for importing.

I can't remember if there was a stable consensus on those principles, especially the last one, but the above reflect my gut feeling as of today.